components/Button.tsx

```diff
@@ -19,14 +21,15 @@ const Button = (props: PropsWithChildren<{
 			props.onClick(event);
 		}
 	}
+	const basicClass = 'inline-flex items-center justify-center relative cursor-pointer align-middle disabled:cursor-default box-border min-w-max py-1 px-4 rounded-md transition-all ';
```

Comment:
Just want to understand why did you not create another variant for this instead of creating an `isNotBasic` flag. Do you think basic buttons can also be text/outlined/contained type?